bundles/binding/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/internal/SqueezeboxBinding.java

```diff
@@ -171,12 +171,12 @@ public void muteChangeEvent(PlayerEvent event) {
 
     @Override
     public void volumeChangeEvent(PlayerEvent event) {
-        numberChangeEvent(event.getPlayerId(), CommandType.VOLUME, event.getPlayer().getVolume());
+        percentChangeEvent(event.getPlayerId(), CommandType.VOLUME, event.getPlayer().getVolume());
     }
 
     @Override
     public void currentPlaylistIndexEvent(PlayerEvent event) {
-        numberChangeEvent(event.getPlayerId(), CommandType.CURRTRACK, event.getPlayer().getCurrentPlaylistIndex());
+        percentChangeEvent(event.getPlayerId(), CommandType.CURRTRACK, event.getPlayer().getCurrentPlaylistIndex());
```

Comment:
I assume you are using PercentType here because it was before and you just renamed the method to be clear, but would this (and others here) be a case where DecimalType ([parent class of PercentType](https://github.com/openhab/openhab1-addons/blob/1.8/bundles/core/org.openhab.core.library/src/main/java/org/openhab/core/library/types/PercentType.java#L20)) is really the more appropriate State?